overview/content-types/shell-code.nix

```diff
@@ -13,7 +13,7 @@ in
       type = types.str;
     };
     input = mkOption {
-      type = types.lines;
+      type = with types; either lines (listOf lines);
```

Comment:
As commented prior, the multiplicity should be handled on the outside. Multiple commands could be a "session", but one command can only be `lines` once.

Reply:
Yes, we've yet to reach that part as we spent most of the time today debugging issues with the copy buttons themselves (which are now solved):

  - the `Debian 12 and Ubuntu 24.04/24.10` section had 2 copy buttons
  - after you copy for the first time, you can't copy again

But I agree with you that this should be one command.